xds/bootstrap/credentials.go

```diff
@@ -68,3 +70,15 @@ func (d *googleDefaultCredsBuilder) Build(json.RawMessage) (credentials.Bundle,
 func (d *googleDefaultCredsBuilder) Name() string {
 	return "google_default"
 }
+
+// jwtCallCredsBuilder implements the `Credentials` interface defined in
+// package `xds/bootstrap` and encapsulates JWT call credentials.
+type jwtCallCredsBuilder struct{}
+
+func (j *jwtCallCredsBuilder) Build(configJSON json.RawMessage) (credentials.Bundle, func(), error) {
```

Comment:
This change also makes it clear that we should have a separate interface for channel credentials. There is no reason to return a bundle when building channel credentials. Please let me know if you thought there was a good reason for doing that.